packages/react-dom/src/client/ReactDOMEventHandle.js

```diff
@@ -146,21 +146,21 @@ function registerReactDOMEvent(
     registerEventOnNearestTargetContainer(
       targetFiber,
       topLevelType,
-      passive,
-      priority,
-      capture,
+      isPassiveListener,
+      listenerPriority,
+      isCapturePhaseListener,
     );
   } else if (isValidEventTarget(target)) {
     const eventTarget = ((target: any): EventTarget);
     const listenerMap = getEventListenerMap(eventTarget);
-    listenToTopLevelEvent(
+    listenToNativeEvent(
       topLevelType,
```

Comment:
Should this change to nativeEventType then?

Reply:
Dan's further change to remove the concept `topLevelType` will address this.